update/Pos/Update/Worker.hs

```diff
@@ -40,11 +40,15 @@ usWorkers = (map fst [processNewSlotWorker, checkForUpdateWorker], mempty)
               NewSlotTerminationPolicy "Update.processNewSlot"
         }
     processNewSlotWorker =
+        -- TODO: [CSL-198] Catch and report non-critical errors, as
+        -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
         localOnNewSlotWorker processNewSlotParams $ \s ->
             recoveryCommGuard "processNewSlot in US" $ do
                 logDebug "Updating slot for US..."
                 processNewSlot s
     checkForUpdateWorker =
+        -- TODO: [CSL-198] Catch and report non-critical errors, as
+        -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
AFAIU, this worker already catches exceptions which may occur during update download, so I think nothing should be changed.